frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/install-lso-sc.tsx

```diff
@@ -0,0 +1,150 @@
+import * as React from 'react';
+import { match } from 'react-router';
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
```

Comment:
Why this ?

Reply:
The ts libraries for react-redux hooks is messed up. Needs fix console-wide.